pkg/providers/pricing/pricing_test.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright 2024 The CloudPilot AI Authors.
```

Comment:
Copyright 2025 The CloudPilot AI Authors.

Reply:
Fixed, also changed year to 2025 in copyright template.